lib/ts/Ptr.h

```diff
@@ -111,10 +111,8 @@ template <class T> class Ptr
 
   T *operator->() const { return (m_ptr); }
   T &operator*() const { return (*m_ptr); }
-
   // Making this explicit avoids unwanted conversions.  See https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Safe_bool .
   explicit operator bool() const { return m_ptr != nullptr; }
-
   int
   operator==(const T *p)
   {
```

Comment:
This change is not part of this PR nor was it part of the original diff.  This change was made to specificall fix a clang format problem that must have existed on 7.1.x.